app/views/events/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render "event", collection: @upcoming_events, heading: "Upcoming Events" %>
```

Comment:
This is better. But I wanna suggest this:

```
<div class="centered">
  <h1>Upcoming</h1>
  <%= render "event", collection: @upcoming_events, heading: "Upcoming Events" %>
</div>

<div class="centered">
  <h2>Past Events</h2>
  <%= render "event", collection: @past_events, heading: "Past Events" %>
</div>

```
We still want the headings in the index page.